controller/loan_tracker/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright © 2024, F2FINTECH. ALL RIGHTS RESERVED.
+ *
+ * This software is the confidential information of F2FINTECH., and is licensed as
+ * restricted rights software. The use, reproduction, or disclosure of this software is subject to
+ * restrictions set forth in your license agreement with F2 FINTECH.
+ */
+const LoanTrackingModel = require("../../model/loan_tracking");
+const { formatResponse } = require("../../utility");
+
+const LoanTrackingController = {
+  getLoanTracking: async (req, res) => {
```

Comment:
we have used return new Promise, keep the format same and do not use async/await